internal/commands/logs/foldable.go

```diff
@@ -0,0 +1,37 @@
+package logs
+
+import (
+	"github.com/cirruslabs/echelon"
+)
+
+type FordableLogsRenderer struct {
+	delegate          echelon.LogRendered
```

Comment:
What do you think about calling this `parentRenderer`? This will make inheritance-like relationship more evident.

Reply:
I chose `delegate` because this foldable renderer delegates all the methods and just adds two extra messages when a scope starts/ends.